pandas/tests/tseries/holiday/test_holiday.py

```diff
@@ -187,7 +191,8 @@ def test_holidays_within_dates(holiday, start, expected):
     # holiday was not in the original date range (e.g., 7/4/2015 -> 7/3/2015).
     if isinstance(holiday, str):
         calendar = get_calendar("USFederalHolidayCalendar")
-        holiday = calendar.rule_from_name(holiday)
+        name, holiday = holiday, calendar.rule_from_name(holiday)
```

Comment:
Could you do this in 2 lines? Is a bit confusing

Reply:
@ph